### PR TITLE
update sub folder routing /moped

### PR DESCRIPTION
**Pull request by mateoclarke**
This PR fixes (🤞 ) the bug I introduced where hitting the staging URL (https://moped.austinmobility.io/moped/) shows as a blank white page.

### CODE REVIEW

File: moped-editor/package.json [hunk 1]
```diff
@@ -1,8 +1,9 @@
 {
-  "name": "react-material-dashboard",
-  "author": "DeviasIO",
+  "name": "atd-moped-editor",
+  "author": "ATD Data & Technology Services",
   "licence": "MIT",
```
Comment (sergiogcx) [inline moped-editor/package.json:4]: Not opinionated, but should we change the license type to GPL or something like that? Just out of curiosity

**Comment by sergiogcx** [comment]
My only observation is that there is an image that does not load, not important but I wonder if there are any other resources elsewhere in the code that do not load (CSS, other images, etc):




The `/static` string does not occur too often in the app, but it might be worth refactoring now rather than later:

**Review by sergiogcx — APPROVED** [review]
It works! Thank you so much for working on this! 🙌

**Comment by mateoclarke** [comment]
@sergiogcx good catch on the assets that didn't load. I'll fix and refactor that before merging. 🙏

**Review by mddilley — APPROVED** [review]
Checked out the deploy preview and also locally. I didn't encounter the same image bug that Sergio mentioned, but it definitely worth looking into. Everything looks great to me! 🎉

File: moped-editor/package.json (hunk 1 above)
Comment (mateoclarke) [reply moped-editor/package.json:4]: yeah, I'm also not opinionated but I think the standard me and @johnclary had discussed a while back was just public domain CC0 1.0 Universal. 

ex: https://github.com/cityofaustin/atd-dockless-dataviz/blob/master/LICENSE

Probably something we should get back into the habit of updating across our projects.

Comment (johnclary) [reply moped-editor/package.json]: yep, that's generally what i use but also often forget to add it. i didn't realize the creative commons license [was available in github's pre-configured list](https://choosealicense.com/licenses/cc0-1.0/), but it is now. nice!

**Pull request merged**